code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -110,6 +138,23 @@
 		COMSIG_ATOM_ABSTRACT_ENTERED = PROC_REF(on_entered),
 	)
 	AddElement(/datum/element/connect_loc, loc_connections)
+	var/turf/open/our_turf = loc
+	//on creation we check adjacent turfs for hot spot to start grouping, if surrounding do not have hot spots we create our own
+	for(var/turf/open/to_check as anything in our_turf.atmos_adjacent_turfs)
+		if(to_check.active_hotspot)
```

Comment:
an early continue would be good here (ghil review mentioned it)

Reply:
like if there is no hot spot we continue? i thought it would kust continue to the next iteration if the if wasnt run?

Reply:
it would behave the same but similar to early returns an early continue would reduce the amount of indentation